paddle/fluid/framework/ir/graph_test.cc

```diff
@@ -264,5 +264,140 @@ TEST(GraphTest, TestAttrCopy) {
   ASSERT_FALSE(dst_g.Has(kFloatValue));
 }
 
+TEST(GraphTest, TestMultiBlock) {
+  if (FLAGS_convert_all_blocks) {
```

Comment:
If you would like unit test to run automatically in every CI PR, you should set FLAGS_convert_all_blocks to true here, and set it back to false at the end of this test case.

Reply:
I have modified the codes to ensure GraphTest.TestMultiBlock works.